data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -19,10 +19,21 @@
 
     private AadAuthenticationHelper aadAuthenticationHelper;
     private String clusterUrl;
+    private String originApplicationVersion;
```

Comment:
ClientVersionForTracing

Reply:
It is not resolved.

Reply:
yea idk why it didnt go in the same commit